backend/src/lib/config/env.ts

```diff
@@ -15,7 +15,8 @@ const envSchema = z
     PORT: z.coerce.number().default(4000),
     REDIS_URL: zpStr(z.string()),
     HOST: zpStr(z.string().default("localhost")),
-    DB_CONNECTION_URI: zpStr(z.string().describe("Postgres database conntection string")),
+    DB_CONNECTION_URI: zpStr(z.string().describe("Postgres database connection string")),
+    DB_ROOT_CERT: zpStr(z.string().describe("Postgres database base64-encoded CA cert").optional()),
```

Comment:
nit: maybe rename this to `DB_SSL_CERT`

Reply:
There're multiple type of cert-types possible in the complete implementation so I referenced the naming given in Google's example [here](https://cloud.google.com/sql/docs/postgres/samples/cloud-sql-postgres-knex-connect-tcp-sslcerts).